src/object.d

```diff
@@ -2914,21 +2973,25 @@ auto byKeyValue(T : V[K], K, V)(T* aa) pure nothrow @nogc
 ///
 @safe unittest
 {
+    import std.stdio : writeln;
+
     auto dict = ["k1": 1, "k2": 2];
-    int sum;
+    int sum = 0;
     foreach (e; dict.byKeyValue)
+    {
+        writeln(e.key, ": ", e.value);
         sum += e.value;
+    }
 
     assert(sum == 3);
 }
 
 /***********************************
- * Returns a dynamic array, the elements of which are the keys in the
- * associative array.
+ * Copies the keys of the associative array into a dynamic array and returns it.
```

Comment:
A little awkward wording. Maybe:

"returns a newly allocated dynamic array containing a copy of the keys from the associative array"